applications/tari_console_wallet/src/init/mod.rs

```diff
@@ -206,21 +222,83 @@ pub async fn init_wallet(
             }
         },
     };
-
     let (wallet_backend, transaction_backend, output_manager_backend, contacts_backend) = backends;
 
     debug!(
         target: LOG_TARGET,
         "Databases Initialized. Wallet encrypted? {}.", wallet_encrypted
-    );
-
+    ); // TODO remove after next TestNet
+    if node_identity.is_some() {
```

Comment:
clippy usually picks this up as `if let Some(x) = ... {`

Reply:
Did all clippy comments